cachito/web/validation.py

```diff
@@ -94,3 +95,22 @@ def handle_wrapper(request):
                 raise ValidationError(exception.detail)
 
         return handle_wrapper
+
+
+@jsonschema.draft4_format_checker.checks("datetime")
+def datetime_timestamp_validator(val):
+    """Validate that datetime/timestamp fields have the correct format."""
+    if not isinstance(val, str):
+        raise ValidationError(
+            f"'{val}' is not string type to be evaluated as datetime(ISO 8601 format)/timestamp."
+        )
+
+    try:
+        datetime.datetime.fromisoformat(val)
+    except ValueError:
+        try:
+            datetime.datetime.fromtimestamp(int(val))
```

Comment:
I think we will need to validate this separately from datetime. A timestamp can probably just be validated as a [number](https://swagger.io/docs/specification/data-models/data-types/#numbers) in the OpenAPI spec

Reply:
And if we do it that way, with multiple types of acceptable input for that parameter, we'll also need to enclose it in [oneOf](https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/#oneof)

Reply:
I think I missed the goal here... why are we adding timestamp validation here? Shouldn't datetime cover all use cases?

Reply:
Timestamps can be used for the finished_from, finished_to param of the /request-metrics endpoints, but we can probably drop that. It's more of an internal endpoint (accessible to users, yes, but I don't think anyone will yell if we drop timestamp support there)